### DISCUSSION TITLE
Possibility to drag'n'drop links to the sidebar

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#873] [Ideas]
### What feature would you like to see?

Possibility to drag'n'drop links to Zen sidebar. It would extend sidebar functionality much more than the + icon on the bottom of it.
[Ekrano įrašas_20240813_200204.webm](https://github.com/user-attachments/assets/e8c7bd2b-0633-4d71-941c-7aec25750587)

### What platform are you seeing the problem on?

Linux

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/246_

**Comment by MrAn0nym** [comment]
DnD to the tab list to open the link in a new tab would also be greatly appreciated 😁
